tests/testthat/test-entity-attribute-value.R

```diff
@@ -89,9 +89,47 @@ test_that("basic encoding and decoding of EAV columns works", {
   foo = Surv(1:3, c(0,1,0))
   x = as(foo, "cSurv")
   y = eav_encode_cSurv(x)
-  y1 = c("1+","2 ","3+")
+  y1 = c("1+","2","3+")
```

Comment:
are you sure about this one? as.character on Surv puts these spaces in there, but I think it can vary by the version of the survival package.

Reply:
It works for me like that, but it's probably a difference in the version. I restored the previous value.